Jenkinsfile

```diff
@@ -228,7 +228,9 @@ try {
 
   stage('Quality gate') {
     node {
-      discoverGitReferenceBuild()
+      discoverGitReferenceBuild(
+        referenceJob: 'centreon/master'
```

Comment:
isn't reference job develop ?

Reply:
no cause the reference branch should be the stable branch